contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -112,14 +112,13 @@ kubeadm join --token=ae23dc.faddc87f5a5ab458 <address>
 
 * `kubeadm` connects to the API server address specified over TLS.  As we don't yet have a root certificate to trust, this is an insecure connection and the server certificate is not validated.  `kubeadm` provides no authentication credentials at all.
   * Implementation note: the API server doesn't have to expose a new and special insecure HTTP endpoint.
+  * (D)DoS concern: Before this flow is secure to use/enable publicly (when not bootstrapping), the API Server must support rate-limiting. There are a couple of ways rate-limiting can be implemented to work for this use-case, but defining the rate-limiting flow in detail here is out of scope.
```

Comment:
@jbeda Can you open a kubernetes/kubernetes issue with more details about that and assign someone?